guildwars2/gamedata.json

```diff
@@ -513,6 +513,66 @@
       "perfected_envoy_shoes": 80399
     }
   },
+  "killproofs": {
+    "areas": {
```

Comment:
You could make `areas` into an array like
```json
areas: [
 {"name" : "Spirit Vale", "encounters": []},
 {"name" : "Salvation Pass", "encounters" : []}
```
and so on, then you could drop the `order` and the a bit ugly thing with OrderedDict after some changes

Reply:
Thanks, wanted to do it somehow like this at first, but then I checked how it was solved with cats and forgot that I wanted to do it this way... (it could also be done with the cat list)

Reply:
Good point with the cats